app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Idv::HowToVerifyController < ApplicationController
+  def show
+    @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+  end
+
+  def update
+    if how_to_verify_form_params['selection'] == 'ipp'
+      redirect_to idv_document_capture_url
+    else
+      redirect_to idv_hybrid_handoff_url
+    end
+  end
```

Comment:
Don't think I fully understand -- what would the alternative look like? No form, just look at the param?

fwiw I imagine there'll be some state eventually, so we can have guards where if the user tries to skip ahead or back we can redirect them to whatever page they should be on

Reply:
Team Ada is implementing a [FlowPolicy](https://github.com/18F/identity-idp/pull/9465) that will be merged soon, so you'll want to integrate with that. Your step_info class method will be a lot like the [one in phone_question_controller](https://github.com/18F/identity-idp/blob/d3b1f5185c4f70fcdb0452a131b12bad74e7d82e/app/controllers/idv/phone_question_controller.rb#L37-L47), and we'll have to add this controller as an option from agreement_controller.

The back button now works for the first part of the flow up to DocumentCapture, so you won't want to prevent people from going back to/from your page.

Reply:
I like the `self.enabled?` idea. I think it would work better in FlowPolicy if there were a new attribute in idv_session indicating that the user had made a choice and what it was. We could brainstorm about this. Or maybe this would just work - we'd have to try it out.

Reply:
PR is merged now!

Reply:
Glad to hear it works for you! Thanks for being our first additional implementer.